### DISCUSSION TITLE
unusable UST (Wormhole) (Asset v2)

### DISCUSSION THREAD

**Discussion by OrsolWel** [wormhole-foundation/wormhole#544] [Problems / bugs] [answered]
I have bridge some UST from the Binance Smart Chain on Solana using Wormhole. In the process I was requested to migrate the token (to asset v2). Now I have my token on Solana but are unable to swap them for un other token like Sol for example.

Do somebody know how or where can I swap my ust tokens?

Than
<img width="361" alt="Capture d’écran 2021-10-18 à 07 34 42" src="https://user-images.githubusercontent.com/92707235/137674600-0d222477-74a4-49e8-99f7-fd55847ca2cf.png">
ks

**Comment by sleepycorgi1** [comment, marked as answer]
Hey OrsolWel!
Markets for this will be coming very soon. We'll keep you updated when they're available through our twitter and discord :) .
